### PR TITLE
apply-diagnostic-setting-network-security-group & audit-diagnostic-setting & audit-enabling-diagnostic-logs-data-lake-analytics & data-lake-store-encryption & deploy-threat-detection-SQL-servers & event-hub-diagnostic-logs-audit

### PR DESCRIPTION
**Pull request by prkuma**

### CODE REVIEW

File: samples/built-in-policy/data-lake-shore-encryption/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Enforce encryption on Data Lake Store accounts
```
[the remark this answers is not in the document]
Comment (prkuma) [reply samples/built-in-policy/data-lake-shore-encryption/README.md]: Which folder should this be in?  I wasn't able to figure out where it should go.

File: samples/DataLake/data-lake-shore-encryption/desktop.ini [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[.ShellClassInfo]
+LocalizedResourceName=data-lake-store-encryption
```
Comment (camillemarie) [inline samples/DataLake/data-lake-shore-encryption/desktop.ini]: Hmmm something weird happened here. Looks like Windows changed the display name for the folder without actually renaming it?

Please delete this desktop.ini file and rename the folder. If it does this again, try creating the new folder, move the files, and delete old folder.

File: samples/Monitoring/apply-diagnostic-setting-network-security-group/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+$assignment 
+````
+
+## Try with CLI
+
+````cli
+
+az policy definition create --name 'apply-diagnostic-setting-network-security-group' --display-name 'Apply Diagnostic Settings for Network Security Groups' --description 'This policy automatically deploys diagnostic settings to network security groups.' --rules 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Monitoring/apply-diagnostic-setting-network-security-group/azurepolicy.rules.json' --params 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Monitoring/apply-diagnostic-setting-network-security-group/azurepolicy.parameters.json' --mode Indexed
+
+az policy assignment create --name <assignmentname> --scope <scope> --params "{'storagePrefix':{'value':'<storagePrefix>'}}" --params "{'rgName':{'value': '<rgName>'}}" --policy "apply-diagnostic-setting-network-security-group"
```
Comment (camillemarie) [inline samples/Monitoring/apply-diagnostic-setting-network-security-group/README.md]: All parameters should together in one object, so 
`--params "{ 'storagePrefix': { 'value': '<storagePrefix>' }, 'rgName': { 'value': '<rgName>' } }"`

**Comment by camillemarie** [comment]
@pilor

File: samples/DataLake/data-lake-store-encryption/azurepolicy.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                  "field": "Microsoft.DataLakeStore/accounts/encryptionState",
+                  "equals": "Disabled"
+               }
+            ]
+         },
+         "then": {
+            "effect": "deny"
+         }
+      }
+   },
```
Comment (camillemarie) [inline samples/DataLake/data-lake-store-encryption/azurepolicy.json]: Please remove trailing commas from all azurepolicy.json files:
`    },`

**Pull request merged**